addons/mail/static/src/messaging/entity/message/message.js

```diff
@@ -147,6 +135,9 @@ function MessageFactory({ Entity }) {
                 if ('record_name' in data && data.record_name) {
                     originThreadData.name = data.record_name;
                 }
+                if ('model_name' in data && data.model_name) {
```

Comment:
not really fond of `message_format` using `model` to actually mean `res_model`, so here with new entry I would definitely name it `res_model_name`

Reply:
I don't understand the reasoning here, if the field is `model` it seems inconsistent to have it's `model_name` counterpart be prefixed.

Reply:
I don't know how easy/hard it is to rename `model` to `res_model` in `message_format`, so in doubt I prefer not to change right now. With `model_name` here, however, since it has been introduced by this PR, I would really clarify its name and prefix with `res_`.